tests/dockerfiles/centos8/Dockerfile

```diff
@@ -1,10 +1,13 @@
-FROM centos:8
+FROM quay.io/centos/centos:centos8
```

Comment:
Why would we like to use unofficial images?

Reply:
I wonder about the same ; there is probably a good reason I assume, so maybe a comment above would be good.

Reply:
tl;dr Good idea, I'm going to switch it back as you suggest.

----
And now, a pointless rambling of why I switched it for those curious . . . :)

Quay as you may know is another image registry like Docker Hub, but Quay is managed by Red Hat – Who are also the producers of CentOS.  I figured that the it would be all the more official to pull it from Quay rather than Docker Hub.

However, the CentOS team push the same exact Image IDs (see below) to both repositories – it makes no difference which one you pull from.

```
docker pull centos:8
docker pull quay.io/centos/centos:8

docker image ls --digests
REPOSITORY                           TAG       DIGEST                                                                    IMAGE ID       CREATED         SIZE
centos                               8         sha256:a27fd8080b517143cbbbab9dfb7c8571c40d67d534bbdee55bd6c473f432b177   5d0da3dc9764   6 months ago    231MB
quay.io/centos/centos                8         sha256:ec16f510e8cd1cceeddc6888fcb37c6d6b7c6863fbc70bdbe660a8ccad4bab6a   5d0da3dc9764   6 months ago    231MB
```

All that said, `centos:8` from Docker Hub is plenty official and easier to type and recognize.  I'll revert this change.